pkg/artifactory/resource_artifactory_user.go

```diff
@@ -57,14 +69,14 @@ func resourceArtifactoryUser() *schema.Resource {
 			"disable_ui_access": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  true,
 				Description: "When enabled, this user can only access the system through the REST API." +
 					" This option cannot be set if the user has Admin privileges.",
 			},
 			"internal_password_disabled": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  false,
 				Description: "When enabled, disables the fallback mechanism for using an internal password when " +
 					"external authentication (such as LDAP) is enabled.",
```

Comment:
Please include the default value in the description, as well as in the doc Markdown file.